autogpt_platform/infra/helm/autogpt-server/values.dev.yaml

```diff
@@ -102,7 +102,7 @@ env:
   SUPABASE_SERVICE_ROLE_KEY: ""
   GITHUB_CLIENT_ID: ""
   GITHUB_CLIENT_SECRET: ""
-  FRONTEND_BASE_URL: "https://dev-builder.agpt.co/"
+  PLATFORM_BASE_URL: "https://dev-builder.agpt.co"
```

Comment:
do we need both of these still?

Reply:
yes, `FRONTEND_BASE_URL` is still required for local dev as long as the frontend runs on a different port than the backend

Reply:
FRONTEND_BASE_URL will fallback to PLATFORM_BASE_URL

Reply:
and the extra / on the end isn't used anywhere?

This change is either under-documented or un needed imo

Reply:
Extra / works too I think, it's just unnecessary

Reply:
I'm surprised the extra `/` that's there in `master` doesn't break anything